tests/unit/App.spec.ts

```diff
@@ -1,21 +1,29 @@
-import Vue from 'vue';
 import { mount, Wrapper } from '@vue/test-utils';
 import BootstrapVue from 'bootstrap-vue';
+import Vuex from 'vuex';
+import Vue from 'vue';
 
 // components
 import Home from '@/views/Home.vue';
 import Header from '@/components/Header.vue';
 import Calendar from '@/components/Calendar.vue';
 import Footer from '@/components/Footer.vue';
+import YearPicker from '@/components/YearPicker.vue';
+
+// Store
+import yearStore from '@/store/year';
```

Comment:
The best part about pushing all the logic into Vuex state like how you did here is that now it becomes very testable.

I will take a stab later to add some unit tests for the really nice YearPicker component

good job, @carrotfarmer !